Components/Queues/AsyncQueueConsumer.php

```diff
@@ -39,7 +39,7 @@ class AsyncQueueConsumer extends Service implements ConsumerInterface, LoggerAwa
      * @param QueueManager $manager
      * @param string       $format
      */
-    public function __construct(QueueManager $manager, string $format = 'json')
+    public function __construct($manager, string $format = 'json')
```

Comment:
Is this consumer used with the phpAmqpLib? If not the type hinting QueueManager can be maintained in the code

Reply:
$manager param is typed again